executor/executor_linux.h

```diff
@@ -248,3 +251,26 @@ static feature_t features[] = {
     {"usb", setup_usb},
     {"sysctl", setup_sysctl},
 };
+
+static void setup_machine()
```

Comment:
I don't see this function called anywhere. What am I missing? Is it wrong merge conflict resolution?